pyuvdata/utils.py

```diff
@@ -26,18 +22,51 @@
 e_prime_squared = 6.73949674228e-3
 
 
-if six.PY2:
-    def _str_to_bytes(s):
-        return s
-
-    def _bytes_to_str(b):
-        return b
-else:
-    def _str_to_bytes(s):
-        return s.encode('utf8')
-
-    def _bytes_to_str(b):
-        return b.decode('utf8')
+def _str_to_bytes(s):
```

Comment:
Since they don't provide much unique functionality I think deprecating them seems pretty reasonable. Is there a specific version they should be marked for?

Reply:
what about 
```
"_str_to_bytes is deprecated and will be removed in pyuvdata version ??. "
"For an input string s, this function is a thin wrapper on s.encode('utf8'). "
"The use of encode is preferred over calling this function."
```

Reply:
The version to specify they will be removed in should be 2.2 (2 major versions after the deprecation is done).